src/main.ts

```
@@ -465,6 +476,52 @@ const createSwitcherWindow = (initialMode?: string): BrowserWindow => {
   const hash = initialMode ? `#mode=${initialMode}` : '';
   window.loadURL(SWITCHER_WINDOW_WEBPACK_ENTRY + hash);
 
+  if (appMode === 'normal') {
+    // Restore the last bounds before the window is first shown. Only if
+    // they still land on a display that exists — a window remembered on an
+    // external monitor must not open off-screen after it is unplugged.
+    settings
+      .get('switcher-window-bounds')
+      .then((saved: unknown) => {
+        const b = saved as { x: number; y: number; width: number; height: number } | null;
```

ESLint

[error] Replace `·x:·number;·y:·number;·width:·number;·height:·number` with `⏎··········x:·number;⏎··········y:·number;⏎··········width:·number;⏎··········height:·number;⏎·······`

(prettier/prettier)